skills/gh-address-comments/scripts/ai_triage.py

```
@@ -0,0 +1,175 @@
+        sys.exit(1)
+
+    threads = data.get("review_threads", [])
+    open_threads = [t for t in threads if not t.get("isResolved") and not t.get("isOutdated")]
+
+    if not open_threads:
+        print("✓ No open threads to triage.")
+        return
+
+    pr = data.get("pull_request", {})
+    print(f"Triaging {len(open_threads)} open thread(s) for PR #{pr.get('number', '?')}: {pr.get('title', '')}")
+    print(f"Sending to Claude...\n")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)